Libraries/CMSIS/Device/Maxim/MAX32570/Include/scn_regs.h

```diff
@@ -78,316 +80,315 @@ extern "C" {
  * @ingroup     scn
  * @defgroup    scn_registers SCN_Registers
  * @brief       Registers, Bit Masks and Bit Positions for the SCN Peripheral Module.
- * @details Smart Card Interface.
+ * @details     Smart Card Interface.
  */
 
 /**
  * @ingroup scn_registers
  * Structure type to access the SCN Registers.
  */
 typedef struct {
-    __IO uint32_t SC_CR;                   /**< <tt>\b 0x00:</tt> SCN CR Register */
-    __IO uint32_t SC_SR;                   /**< <tt>\b 0x04:</tt> SCN SR Register */
-    __IO uint32_t SC_PN;                   /**< <tt>\b 0x08:</tt> SCN PN Register */
-    __IO uint32_t SC_ETUR;                 /**< <tt>\b 0x0C:</tt> SCN ETUR Register */
-    __IO uint32_t SC_GTR;                  /**< <tt>\b 0x10:</tt> SCN GTR Register */
-    __IO uint32_t SC_WT0R;                 /**< <tt>\b 0x14:</tt> SCN WT0R Register */
-    __IO uint32_t SC_WT1R;                 /**< <tt>\b 0x18:</tt> SCN WT1R Register */
-    __IO uint32_t SC_IER;                  /**< <tt>\b 0x1C:</tt> SCN IER Register */
-    __IO uint32_t SC_ISR;                  /**< <tt>\b 0x20:</tt> SCN ISR Register */
-    __IO uint32_t SC_TXR;                  /**< <tt>\b 0x24:</tt> SCN TXR Register */
-    __IO uint32_t SC_RXR;                  /**< <tt>\b 0x28:</tt> SCN RXR Register */
-    __IO uint32_t SC_CCR;                  /**< <tt>\b 0x2C:</tt> SCN CCR Register */
+    __IO uint32_t cr;                   /**< <tt>\b 0x00:</tt> SCN CR Register */
+    __IO uint32_t sr;                   /**< <tt>\b 0x04:</tt> SCN SR Register */
+    __IO uint32_t pn;                   /**< <tt>\b 0x08:</tt> SCN PN Register */
+    __IO uint32_t etur;                 /**< <tt>\b 0x0C:</tt> SCN ETUR Register */
+    __IO uint32_t gtr;                  /**< <tt>\b 0x10:</tt> SCN GTR Register */
+    __IO uint32_t wt0r;                 /**< <tt>\b 0x14:</tt> SCN WT0R Register */
+    __IO uint32_t wt1r;                 /**< <tt>\b 0x18:</tt> SCN WT1R Register */
+    __IO uint32_t ier;                  /**< <tt>\b 0x1C:</tt> SCN IER Register */
+    __IO uint32_t isr;                  /**< <tt>\b 0x20:</tt> SCN ISR Register */
+    __IO uint32_t txr;                  /**< <tt>\b 0x24:</tt> SCN TXR Register */
+    __IO uint32_t rxr;                  /**< <tt>\b 0x28:</tt> SCN RXR Register */
+    __IO uint32_t ccr;                  /**< <tt>\b 0x2C:</tt> SCN CCR Register */
```

Comment:
I expect most customers are using the EMV Contact Library (L1) to access the smart card peripheral.  So, I expect this change will be transparent to them.   But @mbayraker should be the authority on this.

Reply:
I also notice that this PR removed sc_regs.h. Why did we have duplicate files for the Smart Card?

Reply:
@sihyung-maxim The EMV Contact L1 library has its own sc_regs.h file and the library is using a union approach for bitwise operations. The format is very different. It also supports multiple secure micros, not just ME13. When we took over the library, we continued with the same format. The reason was to continue compliance for the older MMLSDK chips. Otherwise, it would have required re-certification. So, we should not change the EMV L1 library's sc_regs.h file.

This MSDK register file won’t affect the library, so it’s okay to update. The ME13 examples could be updated as well if there is any usage of it.

Reply:
Got it, thanks you two! The main reason why I updated scn_regs.h was because someone has asked for the smart card registers in max32570.svd, so they could see them in their debuggers' register view window. I will continue on with the merge since the EMV L1 Library's smart card registers are independent of the MSDK's scn_regs.h.

@jdk-maxim The sc_regs.h file was recently added for Faisal to add in IAR, so this file shouldn't be present in any MSDK release. It was added by mistake. The scn_regs.h file was what the ME13 originally had (present in current MSDK releases).